client/android/app/build.gradle

```diff
@@ -282,6 +282,12 @@ android {
 }
 
 dependencies {
+    // Import the BoM for the Firebase platform
+    implementation platform('com.google.firebase:firebase-bom:30.3.0')
+
+    implementation 'com.google.firebase:firebase-auth'
+    implementation 'com.google.firebase:firebase-firestore'
```

Comment:
I'm pretty sure that these lines are not needed as React Native Firebase is auto linked including the Firebase native lib. Do you reckon why this was added?

Reply:
Probably from one of firebase docs I checked while trying to understand how things work and make it work on my machine

But it was probably a native code documentation, React Native Firebase doesn't mentions it, could probably be removed 👍